### ISSUE TITLE
[Question]

### ISSUE THREAD

**Issue by AminGR** [harvester/harvester#4356]
![image](https://github.com/harvester/harvester/assets/33826995/d6f8d4ad-1b17-4bf9-97de-f45a25af7ce6)

I have installed Harvester/rancher from ISO file and the cluster tools are still not available.
the above screenshot shows that the "Containers with unready status: [httpd] "..
your help/advice would be appreciated.

**Comment by guangbochen** [comment]
I think u may hit the image load timeout issue https://github.com/harvester/harvester/issues/2651#issuecomment-1641386407, please check if you can host the ISO image locally with faster network speed, this problem will get addressed in the upcoming new v1.2.0 release.

**Comment by AminGR** [comment]
Thanks for your reply, but I assume this is not the issue based on following results:
![image](https://github.com/harvester/harvester/assets/33826995/38745c37-a60f-46e8-a64a-19a001e47c48)

**Comment by starbops** [comment]
Hi @AminGR, could you tell us what environment you use to deploy Harvester? Is it a bare-metal server or a VM? Also, the specs of the machines, especially the disk size, are also of interest.

From the screenshot it seems the container image for the harvester-cluster-repo Pod is missing. Could you help us verify if the image exists on the node with the following command?

```
# Need to execute it on the Harvester nodes
$ sudo -i
$ ctr -n k8s.io i ls | grep harvester-cluster-repo
```

And if you could access the Harvester dashboard, please help collect the support bundle file for us to investigate the issue further. Thank you.

**Comment by AminGR** [comment]
Hi @starbops, thanks for your reply.

Tested environments was as follow:
bare-metal installations on : HPE DL360P Gen10 / HPE DL360P Gen9 / HPE DL360P Gen8
Virtualization on ESXi 7.x ( I will give it a shot on Proxmox soon )

"_ctr -n k8s.io i ls | grep harvester-cluster-repo_"    returns nothing

I have tried the installation in none-UEFI mode/ GPT and MBR as well, and it was the same.
The only successful scenario for me (until now) was the deployment on Ubuntu which worked like a charm.

Thanks again!
